games-action/heartofthealien/heartofthealien-1.2.2.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Heart of Alien Redux"
+DESCRIPTION="Heart of The Alien is the sequel for the undoubtably the best adventure of all times, 'Another World', developed back in 1994 by Interplay exclusively for the Sega CD console. This is an open-source project which aims to bring this highly-addictive game to all modern consoles and computer platforms."
```

Comment:
It's still one long line. You need to add a line break at the limit of 80 letters.
You have to escape that linebreak with a "` \`". Mind the space in front of the slash. If there isn't one, words will appear written together when viewed in HaikuDepot. That's why I suggested to re-build and see if it's OK.
But first you have to insert those linebreaks.

Reply:
0X0D
0X0A
INSERTED.

Reply:
Still missing the escaping "\" before the line breaks.

Reply:
that should say "\\"

Reply:
Now correctly?

Reply:
Almost. Should look like this:

DESCRIPTION="Heart of The Alien is the sequel for the undoubtably \\
the bestadventure of all times, 'Another World',  developed back \\
in 1994 by Interplay exclusively for the Sega CD console.

This is an open-source project which aims to bring this \\
highly-addictive game to all modern consoles and computer platforms."

Reply:
Edited.